assets/i18n/en.toml

```
@@ -80,24 +80,24 @@ vehicles = "Parking"
 noEnergy = "No meter data"
 homePower = "Consumption"
 pvProduction = "Production"
-loadpoints = "Loadpoint | Loadpoint | {count} Loadpoints"
+loadpoints = "Charging point | Charging point | {count} charging points"
 battery = "Battery"
-batteryCharge = "Battery charge"
-batteryDischarge = "Battery discharge"
-gridImport = "Grid import"
-selfConsumption = "Self consumption"
-pvExport = "Grid export"
+batteryCharge = "Battery charging"
+batteryDischarge = "Battery discharging"
+gridImport = "Grid use"
```

"Grid use" and "Sent to grid" is attempting to make it a bit easier.
Not sure if that works better.

---

I think `export` / `import` is universally clear in multiple languages, while `Grid use` doesn't tell me much. How is the grid used? For import or export? :DD

---

```suggestion
gridImport = "From the grid"
```
Fair point. I was thinking the idea is to use as little as possible.
"importing" is technically correct, but hard to apply to something other than a conceptual understanding.

---

I picked "Grid use" and "Grid export", since "Send to Grid" looked a little bit odd to me. Especially compared to the other terms.